pymongo/uri_parser.py

```diff
@@ -35,25 +40,48 @@
 DEFAULT_PORT = 27017
 
 
+def _unquoted_percent(s):
+    """Check for unescaped percent signs
+
+    :Paramaters:
+        - `s`: A string. `s` can have things like '%25', '%2525', '%E2%85%A8'
+               and 'Ⅸ' but cannot have unquoted percent like '%foo'.
```

Comment:
Please remove the `'Ⅸ'` because it changes the encoding of this file.

Reply:
Done.